kbfsmd/key_bundle.go

```diff
@@ -0,0 +1,252 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
```

Comment:
I don't know why this wasn't picked up as a rename

Reply:
FWIW you can force it by doing `git mv foo/file bar/file`

Reply:
This file in particular would be valuable as a move because `crypto` has been removed from most of the methods.

Reply:
I tried making a separate commit with just `git mv` but that didn't work either. I think that GitHub just uses the default git heuristic for detecting renames and `key_bundle.go` changed enough to make that heuristic fail.

Reply:
Ah, maybe it's because there's still a file in libkbfs named `key_bundle.go`. Let me try renaming that...

Reply:
That worked! 🎊